app/db.py

```diff
@@ -60,63 +63,76 @@ class OHLCV(Base):
 
 def symbol_exists(symbol):
     """ check if the stock symbol exists in the database """
-    return session.query(exists().where(OHLCV.symbol==symbol))[0]
+    with session_scope() as session:
+        return session.query(
+            exists()
+            .where(
+                OHLCV.symbol==symbol,
+            )
+        )[0][0]
```

Comment:
i was using this function, but replaced it with a different method so now this isnt being used. i guess i will delete it. 😢